File: BasicProgramming.Tests/src/Reverside/Training/BP/P01/ProgramTest.cs
using System;
using Xunit;

namespace Reverside.Training.BP.P01
{
    public class ProgramTest
    {
        [Fact]
        public void Test1()
        {
            Program program = new Program();
            int result = program.Add(2, 3);
            Assert.Equal(5, result);
        }
    }
}

File: src/test/java/za/co/reverside/training/pattern/p01/ProgramTest.java
package za.co.reverside.training.pattern.p01;

import org.junit.Test;

import static org.junit.Assert.*;

public class ProgramTest {

    @Test
    public void test1(){
        Program program = new Program();
        String result = program.generateSquareStarPattern(5);
        assertEquals(
                "*****\n" +
                "*****\n" +
                "*****\n" +
                "*****\n" +
                "*****\n", result);
    }

}